mage_ai/io/snowflake.py

```diff
@@ -172,6 +176,10 @@ def load(
 
         query_string = self._clean_query(query_string)
 
+        if not table_name and not full_table_name:
+            # Try to find table name via query
+            table_name = table_name_parts_from_query(query_string)[2]
```

Comment:
Please firstly check whether the return value of table_name_parts_from_query is None.

Reply:
Added check on c92a9a19635f3ec527768bebdfa3cc8a4d55613f